draft-agntcy-ads.md

```diff
@@ -218,22 +239,33 @@ queries and capability matching across all variants within a given category.
 
 The architecture supports federation across multiple registry instances, enabling:
 
-- **Organizational boundaries**: Different organizations can maintain their own registries while participating in the global directory
-- **Geographic distribution**: Content can be replicated to registries closer to consumers, reducing latency
-- **Specialization**: Registries can focus on specific domains (e.g., medical AI agents, financial analysis tools)
-- **Redundancy**: Critical agent records can be replicated across multiple registries for availability
+- **Organizational boundaries**: Different organizations can maintain their own
+registries while participating in the global directory
+
+- **Geographic distribution**: Content can be replicated to registries closer to
+consumers, reducing latency
+
+- **Specialization**: Registries can focus on specific domains (e.g., medical AI
+agents, financial analysis tools)
+
+- **Redundancy**: Critical agent records can be replicated across multiple
+registries for availability
 
 # MAS Data Discovery
 
-ADS implements a two-level mapping system that enables efficient discovery of Multi-Agent System components through a distributed hash table (DHT) architecture. This approach separates capability-based discovery from content location, providing both scalability and flexibility in agent retrieval.
+ADS implements a two-level mapping system that enables efficient discovery of
+Multi-Agent System components through a distributed hash table {{DHT}}
+architecture. This approach separates capability-based discovery from content
+location, providing both scalability and flexibility in agent retrieval.
 
 ## Two-Level Mapping Architecture
 
 The discovery system operates through two distinct mapping layers:
 
 ### Skills-to-CID Mapping
 
-The first level maps agent capabilities and skills to their corresponding Content Identifiers (CID):
+The first level maps agent capabilities and skills to their corresponding
```

Comment:
I see that below we are still using digests while we are actually using general-purpose CIDs utilising https://github.com/multiformats/cid.

This also relates to the OCI layer. While it is true that we rely on blob hashes that digest identified, there are also CID tags that are the envelope of the digest.